plotting/plottingUtils/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+
+## build the base level library targets
+foreach(lib
+    inputManager
+    styleManager
+    plottingUtils
+    )
+    add_library( ${lib} ${lib}.cpp )
+    target_link_libraries( ${lib} ${ROOT_LIBRARIES} MaCh3::Manager )
+
```

Comment:
Isn't better to MaCh3::All instead of MaCh3::Manager?

Reply:
I wasn't sure. plotting stuff only uses things from manager (only yaml helper and logging) so linking manager seemed more "accurate". 
But also going forward, if in future we wanted to use any plotting utility in any of the other core code e.g. to make some kind of diagnostic/debug plot I think using MaCh3::All here wouldn't allow that right? (I am far from a CMake expert so not sure what's best)

Reply:
I see. So you think in the future plotting could become part of MaCh3::All, it is not right now thus got confused.

I can see your input manager to be used by other fucntions etc.

I was thinking that plotting utils might use parts of [Structs.h](https://github.com/mach3-software/MaCh3/blob/develop/samplePDF/Structs.h) or [StatisticalUtils.h](https://github.com/mach3-software/MaCh3/blob/develop/mcmc/StatisticalUtils.h) for example

Let's leave it unresolved for now then